lang/cs/StyleCop.MSBuild.Targets

```diff
@@ -0,0 +1,142 @@
+﻿<!--
```

Comment:
Where does this file come from? It looks like it should be part of the StyleCop nuget?